portal/Makefile.am

```diff
@@ -40,6 +40,7 @@ nobase_dist_svcdata_DATA = \
 	db/postgresql/update-6.sql \
 	db/postgresql/update-7.sql \
 	db/postgresql/update-8.sql \
+	db/postgresql/update-9.sql \
```

Comment:
update-10 needs to be included in the Makefile. Include update-9 as well, that was missed when update-9.sql was originally created.